src/components/chat/Chat.jsx

```diff
@@ -1,28 +1,66 @@
 import React from 'react';
 import Contacts from './Contacts.jsx';
 import Conversation from './Conversation.jsx';
+import ChooseMe from './ChooseMe.jsx';
+import ChatApi from './ChatApi.js';
 
 export default class Chat extends React.Component {
-  constructor (props) {
-    super(props);
+  constructor () {
+    super();
     this.openConversation = this.openConversation.bind(this);
+    this.selectMe = this.selectMe.bind(this);
     this.state = {
-      currentContact: null
+      currentContact: null,
+      me: null,
+      students: []
     };
+
+    ChatApi.getStudents().then((students) => {
+      this.setState({
+        students: students,
+        me: null
+      });
+    }).catch((err) => {
+      console.log(err);
+    });
   }
+
   openConversation (contact) {
     this.setState({
       currentContact: contact
     });
   }
 
+  selectMe (currentUser) {
+    this.setState({
+      me: currentUser
+    });
+  }
+
   render () {
+    const currentContactActif = this.state.currentContact !== null;
+    const isMe = this.state.me === null;
```

Comment:
Ce nom de variable n'est pas clair.
Si `this.state.me === null` est vrai, alors `isMe` est vrai ? Ca ressemble à un contre-sens.
Pourquoi ne pas plutôt écrire `const isIdentified = this.state.me !== null;` ?